xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_de.properties

```diff
@@ -4967,7 +4967,7 @@ core.viewers.diff.metadata.language=Sprache
 core.viewers.diff.metadata.name=Name
 core.viewers.diff.metadata.web=Bereich
 core.viewers.diff.metadata.space=Bereich
-core.viewers.diff.metadata.enforceRequiredRights=Erforderliche Rechte anwenden
+core.viewers.diff.metadata.enforceRequiredRights=Erforderliche Rechte durchsetzen
```

Comment:
I think I would prefer "anwenden" instead of "durchsetzen". Any reason why you changed this @SimpelMe?

Reply:
"anwenden" would be better. I'm not aware that I changed it from "anwenden". As far as I remember the key I saw in weblate was empty. I guess weblate and branches has some flaws.